### DISCUSSION TITLE
Running both vite_ruby and webpacker in the same app.

### DISCUSSION THREAD

**Discussion by meenie** [ElMassimo/vite_ruby#295] [Q&A] [answered]
Greetings,

First off, thanks a ton for this gem! It has/will help(ed) us immensely!

I'm trying to introduce this one pack at a time, but am finding that when `bin/rails assets:precompile` runs that it no longer runs webpacker to build that part of the application.  Is this something that should just work?

**Comment by ElMassimo** [comment, marked as answer]
Hi Cody!

Yes, this is something that should just work (haven't heard of this issue before). Normally `assets:precompile` would continue to run as usual, as it's [extended if defined](https://github.com/ElMassimo/vite_ruby/blob/main/vite_ruby/lib/tasks/vite.rake#L56), not overriden.

I'd suggest [debugging the rake tasks for Vite Ruby and Webpacker](https://maximomussini.com/posts/debugging-ruby-libraries), to understand what might be causing this behavior in your setup.

You can also try [disabling the extension of `assets:precompile`](https://vite-ruby.netlify.app/guide/deployment.html#disabling-extension-of-the-assets-precompile-task) and run `vite:build` manually if that's preferable in your setup.

**Comment by meenie** [comment]
Thank you!

**Reply by meenie** [reply]
@ElMassimo, thank you for getting back to us so quickly!  I think it's something that we are doing within our app. I'll let you know when we figure it out.
